ovos_dinkum_listener/service.py

```diff
@@ -791,11 +794,14 @@ def _handle_listen(self, message: Message):
                 if not self.config["listener"].get("instant_listen"):
                     self.voice_loop.state = ListeningState.CONFIRMATION
                     self.voice_loop.confirmation_event.clear()
-                    Timer(0.5, lambda: self.voice_loop.confirmation_event.set()).start()
+                    if not self.voice_loop.confirmation_event.wait(0.5):
+                        LOG.warning("Confirmation event timed out!")
+                        self.voice_loop.confirmation_event.set()
```

Comment:
if this is added here, then doesnt make sense here https://github.com/OpenVoiceOS/ovos-dinkum-listener/blob/ad0954dff1da596eca9975bfb12960fe3e8eb8dd/ovos_dinkum_listener/voice_loop/voice_loop.py#L250

i think the idea is to allow the loop to continue uninterrupted instead of blocking for 0.5 seconds, as this event is supposed to come from an external source (ovos-audio)

i would suggest making the timeout configurable
 
```python
if self.chunk_callback is not None:
    self._chunk_info.energy = self.debiased_energy(chunk, self.mic.sample_width)
    self.chunk_callback(self._chunk_info)
```

 
 @emphasize should be more familiar with this part of the code and give a better rationale on why it was done this way

Reply:
this affects usage of get_response via voice satellites, that was the main reason for introducing this new state in the loop, as get_response would often timeout just because the sat device started listening too late, 

if the `instant_listening` flag is not set then it's supposed to ensure audio is only recorded once the acknowledgment sound is played